router/index.js

```diff
@@ -1,47 +1,103 @@
 'use strict';
 
 const express = require('express');
+const db = require('../server/db');
 
 const router = express.Router();
 
+// Home route
+router.get('/', async (req, res, next) => {
+  try {
+    // Get all the sun shade porducts json
+    const sunshades = await db.getSunshades();
+
+    // Send status and render home view
+    res
+      .status(200)
+      .render('home', {
+        pageId: 'home',
+        title: 'Home',
+        sunshades: sunshades.map(sunglasses => ({
+          ...sunglasses,
+        })),
+      });
+  } catch (error) {
+    next(error);
+  }
+});
+
 // Register route
-router.get('/register', (req, res) => {
+router.get('/register', (req, res, next) => {
```

Comment:
Next is implemented in anticipation of the part of project. Although this is an empty route it will be expanded shortly so good job preparing it for the next portion.